src/io/procops.c

```diff
@@ -737,7 +737,7 @@ static void spawn_setup(MVMThreadContext *tc, uv_loop_t *loop, MVMObject *async_
             MVMObject *error_cb;
             MVMString *msg_str;
 
-            snprintf(error_str, 127, "Failed to spawn process %s: %s (error code %d)",
+            snprintf(error_str, 127, "Failed to spawn process %s: %s (error code %ld)",
```

Comment:
given that `uv_spawn()` returns an `int` (cf [uv.h](https://github.com/libuv/libuv/blob/c4e9657d59723a6273ce68bda935680e1b3596c5/include/uv.h#L1000)), personally I'd adjust the declaration of `spawn_result` instead...

Reply:
I agree -- fixed that & changed the type.